observations/obs_converters/ocean_color/README.rst

```diff
@@ -0,0 +1,96 @@
+and create DART observation sequence (obs_seq) files. All programs are built in the ``ocean_color/work``
+directory and can be done by running ``./quickbuild.csh`` In addition to the main converter (i.e.,
+``convert_sat_chl``), other programs such ``advance_time`` and ``preprocess`` utilities will be built.
+
+**Converter namelist** ``convert_sat_chl_nml``:
+This namelist is added to the rest of DART program namelists in file ``input.nml``. Namelists start
+with an ampersand '&' and terminate with a slash '/'.
+
+::
+
+   &convert_sat_chl_nml
+      file_in         = '../data/V2020350.L3m_DAY_SNPP_CHL_chlor_a_4km.nc'
+      file_out        = '../data/obs_seq_chl'
+      chl_thresh      = 0.03
+      subsample_intv  = 1
+      special_mask    = .true.
+      debug           = .false.
+      /
+
+.. container::
+
+  +-----------------+-----------+-----------------------------------------------------------------------------------------+
+  | Contents        | Type      | Description                                                                             |
+  +=================+===========+=========================================================================================+
+  | file_in         | character | Name of the input netcdf data file. Example of absolute path:                           |
+  |                 |           | ``$DART/observations/obs_converters/ocean_color/data/V2020336.L3m_DAY_SNPP_CHL_4km.nc`` |
+  +-----------------+-----------+-----------------------------------------------------------------------------------------+
+  | file_out        | character | Partial filename for the output file.  The date and time are appended to ``file_out``   |
+  |                 |           | to construct a unique filename reflecting the time of the observations in the file.     |
+  +-----------------+-----------+-----------------------------------------------------------------------------------------+
+  | chl_thresh      | real(r8)  | When the observed chlorophyll values are small, a threshold value is used for the obs.  |
+  |                 |           | Example: ``chl_thresh = 0.03``                                                          |
+  +-----------------+-----------+-----------------------------------------------------------------------------------------+
+  | subsample_intv  | integer   | It is possible to 'thin' the observations. ``subsample_intv``                           |
+  |                 |           | allows one to take every nth observation.                                               |
+  +-----------------+-----------+-----------------------------------------------------------------------------------------+
+  | special_mask    | logical   | A simple procedure to ignore data in certain areas.                                     |
+  |                 |           | User can edit this according to their case.                                             |
```

Comment:
The user has to edit the code in convert_sat_chl.f90  (~line 90) to change the mask. 

```diff
- User can edit this according to their case.
+Users can edit the code in convert_sat_chl.f90 to change the mask location.
```

Reply:
fixed in 48ebf7fba761911a95de89d4653158eebcd752d8